### ISSUE TITLE
Support more loader options

### ISSUE THREAD

**Issue by postb99** [AurityLab/recaptcha-v3#4]
Hi,

According to https://developers.google.com/recaptcha/docs/invisible#render_param I guess that I can pass some options to the "render" parameter. The doc is for invisible captcha V2 but it seems applicable to V3 too.

I'm particularly interested by setting data-badge="inline" so that I can position the logo by CSS, but not hide it. (https://stackoverflow.com/questions/42845324/invisible-recaptcha-badge-positioning-issue)

Could your loader interface accept more flexible parameters and pass then to the recaptcha call as render param ?

https://github.com/AurityLab/recaptcha-v3/blob/c5a5416028ee6c6ae4627bf946c599ce65735b11/src/ReCaptchaLoader.ts#L121

In fact I use vue-recaptcha-v3 so you would need to update both packages.

Thanks, hoping my request is clear enough? I'm new to recaptcha :)

EDIT : I managed adding simple css rule for grecaptcha-badge class and it looks ok in my design, but it's unclear whether we're able to customize the badge rendering language with V3. What do you think? Where could I ask?

**Comment by WipeAir** [comment]
Hey,
sounds like a good idea :)
I'll look into it this evening and try to provide a new version with this feature this evening or tomorrow.

**Comment by WipeAir** [comment]
I just saw your `EDIT`. 
If you have any specific questions about reCAPTCHA directly I'd suggest one of their official [support channels](https://developers.google.com/recaptcha/docs/faq).

Just to be sure, do you still need the option to add additional parameters?

**Comment by postb99** [comment]
Yes, at least the hl parameter for language 🙂. Thanks

Le ven. 26 avr. 2019 à 16:59, Andre Loreth <notifications@github.com> a
écrit :

**Comment by WipeAir** [comment]
I've added a `renderParameters` option to the loader.
If you want to set the `hl` parameter you have to do the following:
```javascript
load('<site key>', {
  renderParameters: {
    hl: 'de'
  }
}).then((recaptcha) => {
  <...>
})
```

It is available in version `1.5.0` (I've also published a new version of vue-recaptcha-v3)
